### PR TITLE
Contacts modal

### PR DESCRIPTION
**Pull request by KageRyu7**
Removed 'Help' and 'Feedback' links in the Navbar and replaced them with a 'Contact' link to leads to a modal containing all forms of contact.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "lodash-amd": "2.4.1",
     "moment": "2.9.0",
     "object-assign": "1.0.0",
-    "ozp-react-commons": "git://github.com/ozone-development/ozp-react-commons#master",
+    "ozp-react-commons": "git://github.com/ozone-development/ozp-react-commons#contacts-modal",
```
Comment (rpokorny) [inline package.json]: Did you mean to add this to the merge?  Once https://github.com/ozone-development/ozp-react-commons/pull/25 is merged it shouldn't be necessary

Comment (KageRyu7) [inline package.json]: No this shouldn't be added. This will also have to be removed for Hud.

**Comment by wski** [comment]
@KageRyu7 Could you switch out what @rpokorny mentioned (an easy trick to avoid needing to do this is `npm install ../ozp-react-commons` with the branch checked out in your local commons. Also please merge master into this, there are a few big changes that are making it hard to test (mainly our upgrade to node-v0.12.7). After a quick look everything looks good though, but need those upgrades in to fully test.

**Comment by wski** [comment]
Once rebased to one commit in correct format  `feat(Contact Modal): Added contact modal to global navbar` or similar it will be :+1:

**Comment by wski** [comment]
Added a commit to the readme (newline) with the correct formatting since this rebasing is not going well. So now we have a thumbs up!

**Comment by clarkjefcoat** [comment]
:+1:

**Pull request merged**